proxy/lang.go

```diff
@@ -29,13 +29,38 @@ import (
 	"strings"
 )
 
+func getLanguage(lang string) map[string]string {
+	v, ok := languages[lang]
+	if !ok {
+		return nil
+	}
+	parts := strings.Split(lang, "-")
+	if len(parts) == 2 {
+		if base, ok := languages[parts[0]]; ok {
+			out := make(map[string]string)
+			for k, v := range base {
+				out[k] = v
+			}
+			for k, v := range v {
```

Comment:
Please use a different variable name. The v used in different scopes is confusing.

Reply:
You're over correcting. Short variable names are fine. Using the _same_ name to mean different things in the problem.